bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -1301,10 +1301,31 @@ class AudioManager {
     const { outputDeviceId } = this.bridge;
 
     if (outputDeviceId && typeof audioAlert.setSinkId === 'function') {
-      return audioAlert.setSinkId(outputDeviceId).then(() => audioAlert.play());
+      return audioAlert
+        .setSinkId(outputDeviceId)
+        .then(() => this.canAutoplayAudioElement(audioAlert));
     }
 
-    return audioAlert.play();
+    return this.canAutoplayAudioElement(audioAlert);
+  }
+
+  canAutoplayAudioElement(element) {
```

Comment:
```suggestion
  playAudioElement(element) {
```

Function name is misleading. It's not checking whether an element is playable, but trying to play it while catching exceptions and checking for availability.